app/form_models/service_form.rb

```diff
@@ -5,8 +5,10 @@ class ServiceForm
                 :name_en, :description_en, :website_en, :information_en,
                 :name_el, :description_el, :website_el, :information_el
 
-  validates :name_en, :description_en, :website_en, :information_en,
-            :name_el, :description_el, :website_el, :information_el, presence: true
+  validates :name_en, :description_en, :website_en,
+            :name_el, :description_el, :website_el, presence: true
+
+  validates :information_en, :information_el, presence: true, if: -> { topic.present? }
```

Comment:
I don't think this is necessarily valid. We can probably just drop the validation completely.